### PR TITLE
Virtualbox 5.0

### PR DESCRIPTION
**Pull request by TimVNL**
Added virtualbox-5.0 and virtualbox-extension-pack-5.0 versions. 

Because vagrant isn't yes compatible with virtualbox 5.1

### CODE REVIEW

**Comment by vitorgalvao** [comment]
There was a [pull request template](https://github.com/blog/2111-issue-and-pull-request-templates) when you submitted. It must be filled.

If you were not presented a template, you may have submitted via a CLI tool (likely [`hub`](https://hub.github.com/)). It that was the case, you may wish to see if the tool has a way to let you use the template (`hub` [is working on it](https://github.com/github/hub/issues/1118)).

In either case, please [refer to the template](https://raw.githubusercontent.com/caskroom/homebrew-cask/master/.github/PULL_REQUEST_TEMPLATE.md) and update your comment.

**Comment by jawshooah** [comment]
From the readme:

> Refuse legacy versions of gratis or open-source software, unless there is a clear demonstrable need for them.

Virtualbox is currently at version 5.1. What need is there for 5.0?

**Comment by fanquake** [comment]
@jawshooah, it was mentioned in the PR
```
Because Vagrant isn't yes compatible with virtualbox 5.1
```

File: Casks/virtualbox-5.0.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+cask 'virtualbox-5.0' do
+  if MacOS.release <= :lion
+    version '4.3.32-103443'
```
Comment (jawshooah) [inline Casks/virtualbox-5.0.rb]: This cask is specifically for 5.0, why bother with this?

File: Casks/virtualbox-5.0.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    sha256 '08defbf310b7ba5852fa8dd951438bb9b1528bb1544211568861986110e807f7'
+  else
+    version '5.0.22,1081085'
```
Comment (jawshooah) [inline Casks/virtualbox-5.0.rb]: According to the [download page](https://www.virtualbox.org/wiki/Download_Old_Builds_5_0), the latest patch version of 5.0 is 5.0.24. Why use 5.0.22 here?

File: Casks/virtualbox-5.0.rb [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    sha256 '05acd9588bf24441ce6573df36547ba943b22651c73879d8e90603f457d4782e'
+  end
+
+  url "http://download.virtualbox.org/virtualbox/#{version.before_comma}/VirtualBox-#{version.before_comma}-#{version.after_comma}-OSX.dmg"
+  name 'Oracle VirtualBox'
+  homepage 'https://www.virtualbox.org/wiki/Download_Old_Builds_5_0'
+  license :gpl
+
+  auto_updates false
```
Comment (jawshooah) [inline Casks/virtualbox-5.0.rb]: Not needed unless it's `true`.

File: Casks/virtualbox-5.0.rb [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+  pkg 'VirtualBox.pkg'
+  binary '/Applications/VirtualBox.app/Contents/MacOS/VBoxManage'
```
Comment (jawshooah) [inline Casks/virtualbox-5.0.rb]: No need for these, the `pkg` installs them for us.

File: Casks/virtualbox-5.0.rb [hunk 5]
```diff
@@ -0,0 +1,29 @@
+cask 'virtualbox-5.0' do
```
Comment (jawshooah) [inline Casks/virtualbox-5.0.rb]: The [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md#cask-filenames) must be followed. This should be `virtualbox50`, with the file renamed to match.

File: Casks/virtualbox-extension-pack-5.0.rb [hunk 6]
```diff
@@ -0,0 +1,22 @@
+cask 'virtualbox-extension-pack-5.0' do
```
Comment (jawshooah) [inline Casks/virtualbox-extension-pack-5.0.rb]: Should be `virtualbox-extension-pack50`, with the file renamed to match.

File: Casks/virtualbox-extension-pack-5.0.rb [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+  version '5.0.22,108108'
+  sha256 'f671fcdf7d5365fea630127082132bd38067c3afed8b7490c137706046b641a6'
+
+  url "http://download.virtualbox.org/virtualbox/#{version.before_comma}/Oracle_VM_VirtualBox_Extension_Pack-#{version.before_comma}-#{version.after_comma}.vbox-extpack"
+  name 'Oracle VirtualBox Extension Pack'
+  homepage 'https://www.virtualbox.org'
+  license :closed
+
+  depends_on cask: 'virtualbox-5.0'
```
Comment (jawshooah) [inline Casks/virtualbox-extension-pack-5.0.rb]: Should be `caskroom/versions/virtualbox50`

**Comment by fanquake** [comment]
@TimVNL Do you want to make the necessary changes so this can be merged, or is it no longer necessary?

**Comment by TimVNL** [comment]
@fanquake Vagrant now also upgrade there package so it's no longer needed

**Pull request closed without merging**